mirar/pipelines/winter/models/_candidates.py

```diff
@@ -56,6 +58,7 @@ class CandidatesTable(WinterBase):  # pylint: disable=too-few-public-methods
     objectid = Column(VARCHAR(40), nullable=False, unique=False)
     deprecated = Column(Boolean, nullable=False, default=False)
     jd = Column(Float, nullable=False)
+    utctime = Column(DateTime(timezone=True))
```

Comment:
Why is this required? Isn't jd sufficient?

Reply:
This will enable to do easy native time queries using psql e.g:

```
select count(candidid) from candidates where AGE(CURRENT_TIMESTAMP, utctime) < '48 hours';
```

Additionally we can monitor candidates in the dashboard

Reply:
OK